kwave/options/simulation_execution_options.py

```diff
@@ -45,23 +43,14 @@ def __post_init__(self):
 
         if self.binary_name is None:
             if self.is_gpu_simulation:
-                self.binary_name = "kspaceFirstOrder-CUDA" if is_unix() else "kspaceFirstOrder-CUDA.exe"
+                self.binary_name = "kspaceFirstOrder-CUDA"
             else:
-                self.binary_name = "kspaceFirstOrder-OMP" if is_unix() else "kspaceFirstOrder-OMP.exe"
+                self.binary_name = "kspaceFirstOrder-OMP"
 
-        self._is_linux = sys.platform.startswith("linux")
-        self._is_windows = sys.platform.startswith(("win", "cygwin"))
-        self._is_darwin = sys.platform.startswith("darwin")
+        if PLATFORM == "windows":
+            self.binary_name += ".exe"
 
-        if self._is_linux:
-            binary_folder = "linux"
-        elif self._is_windows:
-            binary_folder = "windows"
-        elif self._is_darwin:
-            raise NotImplementedError("k-wave-python is currently unsupported on MacOS.")
```

Comment:
Should we still raise an error or at least warn the user that we do not have a binary for MacOS?

Reply:
Great point. Yes.

Reply:
Actually I think the check is already run here so it's not needed in this PR https://github.com/waltsims/k-wave-python/blob/master/kwave/__init__.py